src/status_im/subs/wallet/wallet.cljs

```diff
@@ -95,6 +95,18 @@
         (map #(assoc-network-preferences-names network-details % test-networks-enabled?))
         (sort-by :position))))
 
+(rf/reg-sub
+ :wallet/active-accounts
+ :<- [:wallet/accounts]
+ (fn [accounts]
+   (remove :watch-only? accounts)))
```

Comment:
@Rende11 - We have an existing sub `wallet/accounts-without-watched-accounts` which does the same. Can we use it?

Reply:
Done